### DISCUSSION TITLE
Unable to boot after installation

### DISCUSSION THREAD

**Discussion by Carthago93** [Security-Onion-Solutions/securityonion#12320] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

other (please provide detail below)

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16GB

### Storage for /

200GB

### Storage for /nsm

200GB

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello everyone,

I followed along the tutorial steps on YouTube by the official security onion channel in order to get started with the evaluation mode.

I'm reaching out for some assistance regarding my recent installation of Security Onion on my VM. Initially, I encountered an issue where only the terminal was visible, and the graphical environment was missing right 
![Screenshot 2024-02-07 122921](https://github.com/Security-Onion-Solutions/securityonion/assets/159256020/1bf4045f-27e0-44a5-bbff-e5486aacd984)
after the installation. In an attempt to troubleshoot, I logged out to see if the graphical environment would appear upon system reboot.

Regrettably, I'm now facing a different problem as my system seems unable to boot anymore, as depicted in the screenshot below. Additionally, I'm still unsure about the initial absence of the graphical environment despite what seemed like a successful installation process.

I've been grappling with these challenges for a few days now, and I'm starting to feel frustrated. Any guidance or support would be greatly appreciated.

Julien

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Those look like kernel messages in your screenshot - https://docs.securityonion.net/en/2.4/console.html. You should be able to login or ssh.

Once logged in, check `sudo so-status`.

**Reply by Carthago93** [reply]
Hi,

I disabled the console messages by using the following command 'sudo dmesg -D' and received the replay as shown in the screenshot. What should I do?

Julien
![Screenshot 2024-02-08 203927](https://github.com/Security-Onion-Solutions/securityonion/assets/159256020/dd09db38-3ce9-4f43-b088-a9ab6813ea5c)

**Reply by cm-ops** [reply]
Are you able to run `sudo so-checkin`? Does it complete without errors?

**Reply by Carthago93** [reply]
No it shows errors as you can see. Please help me out of this..
![Screenshot 2024-02-13 204203](https://github.com/Security-Onion-Solutions/securityonion/assets/159256020/ac911f8b-8d2f-42fd-92c5-a7d4f2105637)

**Reply by samuel809** [reply]
How much uptime do you have prior to running so-status? provide at least 10 mins...
some errors there regarding dns, please check (this is custom to your environment)

**Comment by cm-ops** [comment]
Check your minion file at `/etc/salt/minion`. The error above is looking for the `master hostname 'salt'`, there may have been an issue installing. The master setting should be your node hostname `carthago`.

**Reply by Carthago93** [reply]
I do not see any filename called 'minion'. I get the following when navigating to the salt directory. What should I do to fix this issue in order to get the graphical environment of Security Onion? 
![Screenshot 2024-02-14 222943](https://github.com/Security-Onion-Solutions/securityonion/assets/159256020/37ab7148-8b6b-4d7b-9727-ff6bc03606ca)

**Reply by cm-ops** [reply]
Minion is the filename, try `cat /etc/salt/minion`. Let me know if that returns anything.

**Reply by Carthago93** [reply]
I get nothing in return. Could you please support me through TeamViewer? I haven't made any progress yet and it has been going on for 2 weeks.

**Reply by Carthago93** [reply]
![Screenshot 2024-02-19 213027](https://github.com/Security-Onion-Solutions/securityonion/assets/159256020/acedb29f-918c-407c-b27f-01b05424bdcc)
![Screenshot 2024-02-19 212956](https://github.com/Security-Onion-Solutions/securityonion/assets/159256020/b21e2b23-b7fe-4bee-bad4-f37708c75d83)
![Screenshot 2024-02-19 212857](https://github.com/Security-Onion-Solutions/securityonion/assets/159256020/0b3d8dfe-5f3f-43ac-bcf0-ad5e2d74cf3d)
![Screenshot 2024-02-19 212806](https://github.com/Security-Onion-Solutions/securityonion/assets/159256020/acb47562-6b82-4fb2-b29e-0a2aabef372f)
![Screenshot 2024-02-19 212744](https://github.com/Security-Onion-Solutions/securityonion/assets/159256020/307e0454-28c4-449f-8e00-80e3096b53d5)

**Reply by cm-ops** [reply]
You probably want to rerun setup if your minion file is not present.
